src/app/controllers/GraphController.ts

```diff
@@ -1,19 +1,18 @@
 /**
  * Controller for the graph.
  */
-class GraphController {
-  app: any;
-  constructor(app: any) {
+import { App } from "../App";
+export class GraphController {
+  app: App;
+  constructor(app: App) {
     this.app = app;
     // ADD BINDINGS BELOW
     this.query = this.query.bind(this);
   }
 
   // ADD MUTATION METHODS BELOW
 
-  query(query: string, variables: any = {}) {
+  query(query: string, variables: object = {}) {
```

Comment:
Not sure if this is the right way to represent object in TypeScript FYI, I’m pretty sure it’s not and you have to type out the types from GraphQL